tetrad-gui/src/main/java/edu/cmu/tetradapp/editor/SemImParamsEditor.java

```diff
@@ -108,7 +108,7 @@ public double filter(double value, double oldValue) {
             }
         });
 
-        final DoubleTextField covLowField = new DoubleTextField(params.getDouble("covLow", 0.1),
+        final DoubleTextField covLowField = new DoubleTextField(params.getDouble("covLow", 0.0),
```

Comment:
Instead of hardcoding the default value, can you get it by calling `ParamDescriptions`?

````
params.getDouble("covLow", (Double) ParamDescriptions.getInstance().get("covLow").getDefaultValue())
````

Reply:
The problem is then you can't have two IM's with different settings.

Reply:
Oh I see your point.